copyright.txt

```diff
@@ -1,7 +1,7 @@
 ﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
 
 Copyright by Susumu Yoshida - http://www.mcdodesign.com/
-- address_book_32.png
+- pupilist.png
```

Comment:
this copyright.txt states where we get stuff from, cos the original address book icon was taken from a website. since we made our own image this entire part shld be removed

Reply:
@farisanadia care to advise? or i can merge first then you can make a new commit for it or smt?

Reply:
ok i will update this soon